nylas-sync-startup.sh

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh
-export REPLACE_VARS='$MYSQL_ENV_MYSQL_USERNAME:$MYSQL_ENV_MYSQL_PASSWORD:$MYSQL_PORT_3306_TCP_PORT:$MYSQL_HOST'
```

Comment:
Quite complicated and inconvenient way of config overriding. Nylas application has it's own way: merging two config files. Nylas' way is simplier to use and simplier for automation.